yb-voyager/cmd/importSchemaYugabyteDB.go

```diff
@@ -58,6 +60,36 @@ func importSchemaInternal(exportDir string, importObjectList []string,
 	return nil
 }
 
+func generateAnalyzeReport() (string, error) {
+	//check if schema is already analyzed
+	if schemaIsAnalyzed() {
+		return "", nil
+	}
+
+	msr, err := metaDB.GetMigrationStatusRecord()
+	if err != nil {
+		return "", fmt.Errorf("get migration status record: %w", err)
+	}
+	//11.2-YB-2024.2.1.0-b10
+	splits := strings.Split(importTargetDBVersion, "-")
```

Comment:
Let's pass in the target-db-version into this function instead of relying on global variables.